doc/configuration-v2_1-experimental.md

```diff
@@ -39,6 +39,7 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_mount_** (object): contains the set of mount and formatting options for the filesystem. A non-null entry indicates that the filesystem should be mounted before it is used by Ignition.
       * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
       * **format** (string): the filesystem format (ext4, btrfs, xfs, or swap).
+      * **_wipeFilesystem_** (boolean): if false, and a matching filesystem exists, it will be used. If false, and a non-matching filesystem exists, Ignition will attempt to create the filesystem and may fail. If true, any preexisting filesystem will be destroyed.
```

Comment:
This wording is too confusing. What does "matching" mean? What does "and my fail" mean? Rather than describing the decision tree, just explain that the option controls whether or not the device is cleared (not exactly true) before the filesystem is created.